src/styles/global/body.styl

```diff
@@ -4,3 +4,12 @@ body
   background-color #fff
   margin 0
   padding 0
+
+header
+  color white
+  background #00979D
```

Comment:
Just a minor style preference, but I prefer use of colons in the key-value pairs in stylus even though it's not required. I just find it more readable.